bigquery_etl/glam/generate.py

```diff
@@ -271,6 +271,7 @@ def main():
                 source_table=f"glam_etl.{args.prefix}__clients_histogram_aggregates_v1",
                 **config[args.prefix],
             ),
+            channel=channel_prefixes[args.prefix],
```

Comment:
I added this after realizing the property `channel` in the data is always `*` for these tables (because the table names already contain the channel). The sampling logic must be applied to `Release` only